docs/pages/pmd/userdocs/extending/writing_pmd_rules.md

```diff
@@ -1,330 +0,0 @@
----
-title:  Writing a custom rule
-tags: [extending, userdocs]
-summary: "Learn how to write a custom rule for PMD"
-last_updated: July 3, 2016
-permalink: pmd_userdocs_extending_writing_pmd_rules.html
```

Comment:
I'd keep this page, in case someone has a deep link to https://pmd.github.io/latest/pmd_userdocs_extending_writing_pmd_rules.html 
I'll readd this page and provide links to the new pages you wrote.